### ISSUE TITLE
[Feature Request] Internationalization

### ISSUE THREAD

**Issue by mmmaxi** [beamworks/react-csv-importer#28]
It would be nice to define custom values for the text elements used in this component. Like this different languages can be supported.

**Comment by unframework** [comment]
Agreed, for sure this is something that comes up for a lot of apps. I am still thinking about how to set up a more "hardcore" enterprise-friendly usage pattern that would allow a much more in-depth customization like i18n strings, theming for local design system, etc. Should probably have a $$ tier for some of this stuff actually.

**Comment by jabidof** [comment]
What about defining the labels used in an object, e.g.:
`{
  rawFileContent: "Contenu du fichier",
  previewImport: "Aperçu de l'importation",
  etc.
}`

That would make "react-csv-importer" independent of the internationalization library...

Just my 2 cents.

**Comment by unframework** [comment]
Thanks for the suggestion, @jabidof, I did a bit of research and will follow the example of [Material UI localization](https://material-ui.com/guides/localization/) which is pretty much what you are describing. I can see that this is one of the more pressing priorities for the library to be useful - hoping to get something out within a week or two!

**Comment by jabidof** [comment]
@unframework 
Thanks for the feedback.
FYI, I had to implement my own CSV importer (using AG-Grid as a base) but I'm more than interested in using yours. I see much room in the react ecosystem for such a lib (with a PRO paid version?). I'll keep myself informed about your dev.
While implementing I saw need to define types of data to be imported, save import settings; maybe some features for the future...
Keep in touch.
